packages/app-staking/src/NewStake/index.tsx

```diff
@@ -127,7 +127,7 @@ function NewStake ({ className, isVisible }: Props): React.ReactElement<Props> {
       setOpenHelpDialog(false);
     }
 
-    const openAccountCheckingModal = !hasAccounts || stakedAccounts.length == allAccounts.length || unstakedAccounts.length === 0;
+    const openAccountCheckingModal = !hasAccounts || stakedAccounts.length == allAccounts.length;
```

Comment:
noticed the flickering and removed this bit for now.. will think about proper solution for this